ardb.gemspec

```diff
@@ -18,7 +18,8 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_development_dependency("assert", ["~> 2.15.1"])
+  gem.add_development_dependency("assert", ["~> 2.16.0"])
+  gem.add_development_dependency("pg",     ["~> 0.17.1"])
```

Comment:
@jcredding why not add `pg` to the Gemfile instead?  I'd prefer it not be formally listed as a development dependency.